scripts/github_api

```diff
@@ -71,6 +71,40 @@ def _print_output(output):
         print '%s="%s"' % (k, v)
 
 
+class Status:
+    """
+    https://developer.github.com/v3/repos/statuses/
+    """
+    def __init__(self, repo, reference):
+        """
+        Args:
+            repo (str): target repository path. Format:
+                <repo owner>/<repo>
+            reference (str): git reference. Either a sha1 or a
+                pull-request number
+        """
+        self.ref = reference
+        self._endpoint = os.path.join(
```

Comment:
I'd prefer to not use API-specific terminology, but I have already mentioned this other times. I'll leave it up to you.